package.json

```
@@ -0,0 +1,181 @@
+    "getpass": "^0.1.7",
+    "glob": "^7.1.7",
+    "global-dirs": "^3.0.0",
+    "graceful-fs": "^4.2.8",
+    "har-schema": "^2.0.0",
+    "har-validator": "^5.1.5",
+    "has-flag": "^4.0.0",
+    "http-signature": "^1.2.0",
+    "human-signals": "^1.1.1",
+    "indent-string": "^4.0.0",
+    "inflight": "^1.0.6",
+    "inherits": "^2.0.4",
+    "ini": "^2.0.0",
+    "is-ci": "^3.0.0",
+    "is-fullwidth-code-point": "^3.0.0",
+    "is-installed-globally": "^0.4.0",
+    "is-path-inside": "^3.0.3",
+    "is-stream": "^2.0.1",
+    "is-typedarray": "^1.0.0",
+    "is-unicode-supported": "^0.1.0",
+    "isexe": "^2.0.0",
+    "isstream": "^0.1.2",
+    "jsbn": "^0.1.1",
+    "json-schema": "^0.2.3",
+    "json-schema-traverse": "^0.4.1",
+    "json-stringify-safe": "^5.0.1",
+    "jsonfile": "^6.1.0",
+    "jsprim": "^1.4.1",
+    "lazy-ass": "^1.6.0",
+    "listr2": "^3.11.1",
+    "lodash": "^4.17.21",
+    "lodash.once": "^4.1.1",
+    "log-symbols": "^4.1.0",
+    "log-update": "^4.0.0",
+    "merge-stream": "^2.0.0",
+    "mime-db": "^1.49.0",
+    "mime-types": "^2.1.32",
+    "mimic-fn": "^2.1.0",
+    "minimatch": "^3.0.4",
+    "minimist": "^1.2.5",
+    "ms": "^2.1.3",
+    "npm-run-path": "^4.0.1",
+    "object-assign": "^4.1.1",
+    "once": "^1.4.0",
+    "onetime": "^5.1.2",
+    "ospath": "^1.2.2",
+    "p-map": "^4.0.0",
+    "path-is-absolute": "^1.0.1",
+    "path-key": "^3.1.1",
+    "pend": "^1.2.0",
+    "performance-now": "^2.1.0",
+    "pify": "^2.3.0",
+    "pretty-bytes": "^5.6.0",
+    "psl": "^1.8.0",
+    "pump": "^3.0.0",
+    "punycode": "^2.1.1",
+    "qs": "^6.5.2",
+    "querystring": "^0.2.0",
+    "ramda": "^0.27.1",
+    "request-progress": "^3.0.0",
+    "restore-cursor": "^3.1.0",
+    "rimraf": "^3.0.2",
+    "rxjs": "^6.6.7",
+    "safe-buffer": "^5.2.1",
+    "safer-buffer": "^2.1.2",
+    "shebang-command": "^2.0.0",
+    "shebang-regex": "^3.0.0",
+    "signal-exit": "^3.0.3",
+    "slice-ansi": "^3.0.0",
+    "sshpk": "^1.16.1",
+    "string-width": "^4.2.2",
+    "strip-ansi": "^6.0.0",
+    "strip-final-newline": "^2.0.0",
+    "supports-color": "^8.1.1",
+    "throttleit": "^1.0.0",
+    "through": "^2.3.8",
+    "tmp": "^0.2.1",
+    "tough-cookie": "^2.5.0",
+    "tslib": "^1.14.1",
+    "tunnel-agent": "^0.6.0",
+    "tweetnacl": "^0.14.5",
+    "type-fest": "^0.21.3",
+    "universalify": "^2.0.0",
+    "untildify": "^4.0.0",
+    "uri-js": "^4.4.1",
+    "url": "^0.11.0",
+    "uuid": "^8.3.2",
+    "verror": "^1.10.0",
+    "which": "^2.0.2",
+    "wrap-ansi": "^7.0.0",
+    "wrappy": "^1.0.2",
+    "yauzl": "^2.10.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mbta/reduced_fares.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/mbta/reduced_fares/issues"
+  },
+  "homepage": "https://github.com/mbta/reduced_fares#readme"
```

```suggestion
  }
```

Can we please remove these lines? I don't think they are things we typically want to list (since we won't be publishing this as a library).